Source/VCGeneration/Split.cs

```diff
@@ -949,13 +953,13 @@ bool ContainsAssert(Block b)
                || c is AssertCmd && splitOnEveryAssert;
       }
       
-      public static List<Split /*!*/> FindManualSplits(Split s, bool splitOnEveryAssert)
+      public static List<Split /*!*/> FindManualSplits(Split initialSplit, bool splitOnEveryAssert)
```

Comment:
This and the rest of the changes to this method are somewhat unrelated to the command-line option changes, but they're all semantics-preserving cleanups, so I'm not really opposed to including them in one PR.